src/components/events/EventModal/style.module.scss

```diff
@@ -0,0 +1,108 @@
+@use 'src/styles/vars.scss' as vars;
+
+.modal {
+  background-color: var(--theme-accent-line-1-transparent);
+  border: 0;
+  display: flex;
+  height: 100vh;
+  left: 0;
+  opacity: 0;
+  overscroll-behavior: contain;
+  padding: 2rem;
+  pointer-events: none;
+  position: fixed;
+  top: 0;
+  transition: all 0.2s;
+  user-select: none;
+  visibility: hidden;
+  width: 100vw;
+  z-index: 1000; // Just one above the header's 999
+
+  &[open] {
+    opacity: 1;
+    pointer-events: unset;
+    user-select: text;
+    visibility: visible;
+
+    .modalBody {
+      transform: none;
+    }
+  }
+
+  &:modal {
+    max-height: unset;
+    max-width: unset;
+  }
+
+  &::backdrop {
+    background: none;
+  }
+
+  .modalBody {
+    background-color: var(--theme-background);
+    border-radius: 10px;
+    margin: auto;
+    max-width: 100%;
+    overflow: hidden;
+    transform: scale(0.95);
+    transition: transform 0.2s;
+
+    .image {
+      flex-shrink: 0;
+      height: 11.5rem;
+      overflow: hidden;
+      position: relative;
+    }
+
+    .contents {
+      display: flex;
+      flex-direction: column;
+      gap: 2.5rem;
+      max-width: 70rem;
+      padding: 2.5rem;
+      width: 70vw;
```

Comment:
the modal is very narrow on mobile. this is because of this `width: 70vw` line, but you could also remove the horizontal padding on `.modal` when the screen is narrow enough, so there's more horizontal screen space to work with

current | suggested
--- | ---
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/a9f93848-9852-4b0b-a2e5-0944b9b5e486) | ![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/f205abd0-b27a-4ff4-aff3-378a8a9061c9)

Reply:
fixed !